astropy/cosmology/_src/typing.py

```diff
@@ -1,12 +1,17 @@
 """Static typing for :mod:`astropy.cosmology`. PRIVATE API."""
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-__all__ = ["_CosmoT"]
+__all__ = ["FArray", "_CosmoT"]
 
-from typing import TYPE_CHECKING, TypeVar
+from typing import TYPE_CHECKING, Any, TypeAlias, TypeVar
+
+import numpy as np
+from numpy.typing import NDArray
 
 if TYPE_CHECKING:
     import astropy.cosmology
 
 _CosmoT = TypeVar("_CosmoT", bound="astropy.cosmology.Cosmology")
 """Type variable for :class:`~astropy.cosmology.Cosmology` and subclasses."""
+
+FArray: TypeAlias = NDArray[np.floating[Any]]
```

Comment:
It's not obvious to me if the alias is worth it or if `FArray` is the best name for it. But in any case
```suggestion
FArray: TypeAlias = NDArray[np.floating]
```